lib/configure.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+var debug = require('debug'),
+  log = debug('nemo:log'),
+  confit = require('confit'),
+  _ = require('lodash'),
+  path = require('path'),
+  Promiz = require('./promise'),
+  handlers = require('shortstop-handlers'),
+  yargs = require('yargs'),
+  error = debug('nemo:error');
+
+log.log = console.log.bind(console);
+error.log = console.error.bind(console);
+
+module.exports = function Configure(_basedir, _configOverride) {
+  log('_basedir %s, _configOverride %o', _basedir, _configOverride);
+  var basedir, configOverride;
+  //settle arguments
+  basedir = (arguments.length && typeof arguments[0] === 'string') ? arguments[0] : process.env.nemoBaseDir || undefined;
+  configOverride = (!basedir && arguments.length && typeof arguments[0] === 'object') ? arguments[0] : undefined;
+  configOverride = (!configOverride && arguments.length && arguments[1] && typeof arguments[1] === 'object') ? arguments[1] : configOverride;
+  configOverride = (!configOverride) ? {} : configOverride;
+
+  log('basedir %s, configOverride %o', basedir, configOverride);
+
+  var prom = Promiz();
+
+  //hack because confit doesn't JSON.parse environment variables before merging
```

Comment:
Yes- I was referring to the comment about the `JSON.parse` hack.  Just mentioned it b/c I didn't fully understand the reason for it, but it makes more sense now.  It also sounds like there's a path for removing it in nemo@3.  Thanks for the additional insight...